src/runtime_src/core/tools/xbmgmt2/OO_Hotplug.cpp

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright (C) 2022 Xilinx, Inc
+ * 
+ * Licensed under the Apache License, Version
+ * 2.0 (the "License"). You may not use this file except in
+ * compliance with the License. A copy of the License is located
+ * at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+// ------ I N C L U D E   F I L E S -------------------------------------------
+// Local - Include Files
+#include "OO_Hotplug.h"
+#include "tools/common/XBUtilitiesCore.h"
+#include "tools/common/XBUtilities.h"
+
+#include "core/common/system.h"
+#include "core/common/query_requests.h"
+
+// 3rd Party Library - Include Files
+#include <boost/program_options.hpp>
+#include <boost/algorithm/string.hpp>
+#include <boost/filesystem.hpp>
+namespace po = boost::program_options;
+
+// =============================================================================
+
+// ----- C L A S S   M E T H O D S -------------------------------------------
+
+OO_Hotplug::OO_Hotplug( const std::string &_longName, bool _isHidden )
+    : OptionOptions(_longName, _isHidden, "Perform hotplug for the given device")
+{
+  m_optionsDescription.add_options()
+    ("device,d", boost::program_options::value<decltype(m_devices)>(&m_devices)->multitoken(), "The Bus:Device.Function (e.g., 0000:d8:00.0) device of interest")
```

Comment:
We support only one device, please remove multitoken here

Reply:
Actually, framework support multiple devices as Input. later it will pick only first device. Hence this multitoken is used.